drivers/hmis/app/graphql/mutations/update_external_form_submission.rb

```diff
@@ -8,14 +8,16 @@ module Mutations
   class UpdateExternalFormSubmission < CleanBaseMutation
     argument :input, Types::HmisSchema::ExternalFormSubmissionInput, required: true
     argument :id, ID, required: true
-    argument :project_id, ID, required: false # not required for backwards compatibility
```

Comment:
This is just walking back a change that is only on `pit-external-forms` https://github.com/greenriver/hmis-warehouse/compare/release-133...pit-external-forms?expand=1#diff-d340d41adf901f7723f0a39b77b876ff73100e13027c74cb5a7a666b12d7a2bd